Source/WebCore/page/Page.cpp

```diff
@@ -1365,6 +1365,11 @@ void Page::enableLegacyPrivateBrowsing(bool privateBrowsingEnabled)
     setSessionID(privateBrowsingEnabled ? SessionID::legacyPrivateSessionID() : SessionID::defaultSessionID());
 }
 
+void Page::enableConsoleLogWithPrivateBrowsing(bool consoleLogEnabled)
+{
+    m_allowsConsoleLogWithPrivateBrowsing = consoleLogEnabled;
```

Comment:
This should be a Settings member. Should be simpler to just define it in Settings.in, and access it later through `m_page.settings()`.